examples/demo/gloo/glsl_sandbox_cube.py

```diff
@@ -11,12 +11,15 @@
 from vispy.util.transforms import perspective, translate, rotate
 
 # Force using qt and take QtCore+QtGui from backend module
-try:
-    app_object = app.use_app('pyqt4')
-except Exception:
-    app_object = app.use_app('pyside')
-QtCore = app_object.backend_module.QtCore,
-QtGui = app_object.backend_module.QtGui
+# try:
+#     app_object = app.use_app('pyqt5')
+#     QtCore = app_object.backend_module.QtCore,
+#     QtGui = app_object.backend_module.QtWidgets
+# except Exception:
+#     app_object = app.use_app('pyside')
+#     QtCore = app_object.backend_module.QtCore,
+#     QtGui = app_object.backend_module.QtGui
+from PyQt5 import QtCore, QtWidgets, QtGui
```

Comment:
I'm not familiar enough with the differences between pyqt and pyside to know, but can the commented out section be made to work if we require 'pyside2'? It should have the same modules as PyQt5.

Reply:
ok i'll do that

Reply:
I don't understand: i don't get the "Create pull request button", maybe because i already have one waiting?

Anyway here is the replacement code:
```
# Force using qt and take relevant modules from backend
try:
    app_object = app.use_app('pyqt5')
except Exception:
    app_object = app.use_app('pyside2')
QtCore = app_object.backend_module.QtCore,
QtWidgets = app_object.backend_module.QtWidgets
QtGui = app_object.backend_module.QtGui
```

Reply:
The existing pull request is automatically updated when you push to the related branch.